test/cli.js

```diff
@@ -1,6 +1,15 @@
 const tape = require('tape');
 const spawn = require('tape-spawn');
 const pkg = require('../package.json');
+const solc = require('../index.json');
```

Comment:
This file is using the `CLI`, it shouldn't acquire the version through this I think. It should check `./solcjs --version`.

Reply:
Hm - if I understand this file correctly, the first test checks whether ``./solcjs --version`` matches ``pkg.version`` anyways, so it may make sense to use ``pkg.version`` to choose the DAO directory? However ``pkg.version``, resp. ``version`` in ``../package.json`` is fixed to ``0.4.24``... I'm actually not sure how this will be run for different versions in the end.